api/point/controllers/point.js

```diff
@@ -34,16 +34,39 @@ module.exports = {
     return sanitizeEntity(entity, { model: strapi.models.point });
   },
 
+  async patchBatchPoint(ctx) {
+    let updatedPoints = [];
+    for (const points of ctx.request.body) {
+      const {id, totalPoints, totalSeasonPoints, availablePoints, projectMeetingMinutes} =  points;
+      const newCtx = {
+        params: {
+          id: id
+        },
+        request: {
+          body: {
+            totalPoints: totalPoints,
+            totalSeasonPoints: totalSeasonPoints,
+            availablePoints: availablePoints,
+            projectMeetingMinutes: projectMeetingMinutes
+          }
+        }
+      };
+      let updated = await this.incrementOrDecrementPoint(newCtx);
```

Comment:
need to make sure when there's an error we don't panic. Add the line when there's error, we still continue.